plugins/cost-insights/src/client.ts

```diff
@@ -30,24 +28,21 @@ import {
   MetricData,
   Project,
   ProjectGrowthData,
-  Trendline,
   UnlabeledDataflowData,
 } from '../src/types';
 import {
   ProjectGrowthAlert,
   UnlabeledDataflowAlert,
 } from '../src/utils/alerts';
-import {
-  exclusiveEndDateOf,
-  inclusiveStartDateOf,
-} from '../src/utils/duration';
+import { inclusiveStartDateOf } from '../src/utils/duration';
+import { trendlineOf, changeOf } from './utils/mockData';
 
 type IntervalFields = {
   duration: Duration;
   endDate: string;
 };
 
-function parseIntervals(intervals: string): IntervalFields {
+export function parseIntervals(intervals: string): IntervalFields {
```

Comment:
Any reason to export this? It's just a hacky util function for the ExampleClient, so I'm wary of adding this even if we don't re-export it elsewhere.